src/index.js

```diff
@@ -17,38 +16,32 @@ class Aurelia {
 
   init(config) {
     this.config = config;
-    let bundle = new BundleCommand(program, this.config, this.logger);
-    let init = new InitCommand(program, this.config, this.logger);
-    let newCmd = new NewCommand(program, this.config, this.logger);
-    let generateCmd = new GenerateCommand(program, this.config, this.logger);
-
-    this.commands[bundle.commandId] = bundle;
-    this.commands[init.commandId] = init;
-    this.commands[newCmd.commandId] = newCmd;
-    this.commands[generateCmd.commandId] = generateCmd;
-
+    program.init(config);
+    program.register(BundleCommand);
```

Comment:
@ahmedshuhel I believe it is possible to inject from the registration. I know that I am injecting the config and logger by default, But you are right. This needs to provided from the registration.